docs/content/_index.md

```diff
@@ -31,17 +31,17 @@ information gathered about the network is aggregated in the Island Server and di
 ## Results
 
 The results of running Monkey Agents are:
- - A map which displays how much of the network an attacker can see, what services are accessible and potential propagation routes.
- - A security report, which displays security issues that Monkey Agents discovered and/or exploited.
- - A MITRE ATT&CK report, which displays the information about the ATT&CK techniques that Monkey Agents tried to use.
- - A Zero Trust report, which displays violations of Zero Trust principles that Monkey Agents found.
+ - A map displaying how much of the network an attacker can see, what services are accessible and potential propagation routes.
+ - A security report, displaying security issues that Monkey Agents discovered and/or exploited.
+ - A MITRE ATT&CK report, displaying the information about the ATT&CK techniques that Monkey Agents tried to use.
+ - A Zero Trust report, displaying violations of Zero Trust principles that Monkey Agents found.
 
 A more in-depth description of reports generated can be found in the [reports documentation page]({{< ref "/reports" >}}).
 
 ## Getting Started
 
-If you haven't downloaded Infection Monkey yet you can do so [from our homepage](https://www.akamai.com/infectionmonkey#download). After downloading the Monkey, install it using one of our [setup guides]({{< ref "/setup" >}}), and read our [getting started guide]({{< ref "/usage/getting-started" >}}) for a quick-start on Monkey!
+If you haven't downloaded Infection Monkey yet you can do so [from our homepage](https://www.akamai.com/infectionmonkey#download). After downloading the Monkey, install it using one of our [setup guides]({{< ref "/setup" >}}), and read our [getting started guide]({{< ref "/usage/getting-started" >}}) for a quick-start on Monkey.
```

Comment:
Infection Monkey's brand is fun and quirky. I don't object to changing some exclamations to periods, but let's be careful not to make the copy cold and sterile.